webAO/master.js

```diff
@@ -101,7 +103,7 @@ function onMessage(e) {
 				+ `<a class="button" href="client.html?mode=watch&ip=${args[2]}:${args[3]}${asset}">Watch</a>`
 				+ `<a class="button" href="client.html?mode=join&ip=${args[2]}:${args[3]}${asset}">Join</a></li>`;
 			server_description[i] = args[1];
-			setTimeout(checkOnline(i, args[2] + ":" + args[3]), 100);
```

Comment:
this was pretty useless to begin with because this will make `setTimeout` invoke the return value of `checkOnline` (which is always `undefined`) as if it were a function, so basically it would have done `undefined()` 100ms after it received the server list.

the correct way to do it would have been to wrap the call in a lambda:
```js
setTimeout(() => checkOnline(i, args[2] + ":" + args[3]), 0);
```